splunk/scripts/pre-run.sh

```diff
@@ -21,9 +21,12 @@ done
 # Index the logs required for the tests
 bash /opt/splunk/log_indexing.sh
 
+# Wait an extra 10 seconds to assure completion of logs ingestion (fix flakiness)
+sleep 10
```

Comment:
did we have a loop somewhere to check if the indexing was done in order to resolve this?

Reply:
indexing != copying and ingesting the DB files
Currently this is the most suitable solution I found.

Reply:
Its not a good solution  - you just introduced a 10 second hard-set delay to all CI jobs ever.

How do YOU know that the ingestion is done?

Does Splunk perhaps has a clean shut-down option that ensures indexing and ingestion is done before it shuts down?

Reply:
While I get why it is not the best in terms of CI - all the other checks, that wait for Splunk to give a response - loop with a delay of 5 seconds on each iteration. this is also not the best but it's something we can work with. 

Since what we are doing is a kind of a "hack", I didn't find a  good way to verify it. maybe only looping (again) to see if `stats count` return anything other than 0 (as it seems like it either ingest all or nothing, this can be an indicator). 

as for the last suggestion, no, I did not find something like this, and due to the nature of this change I'm not sure it'll work even if it does exist.

Reply:
Actually - Speaking of loops, I think the bug resides in the log indexing verification process - it seems like it broke the loop when the check returned False, instead of True. No need for extra wait time,  fixed.

Reply:
See , you just saved your future self a lot of seconds...